### ISSUE TITLE
A better property list organisation in the manpage

### ISSUE THREAD

**Issue by SSDpreowner** [openzfs/zfs#11155]
### System information

Type | Version/Name
 --- | ---
ZFS Version	| 0.8 onwards

### Describe the problem you're observing

In the manpage describing properties (zfs on 0.8 and zfsprops in the master branch) there is a paragraph starting with
`The following three properties cannot be changed after the file system is created`. It doesn't include `encryption`, `keyformat` and `volblocksize`. Wouldn't it be better to gather all the creation time properties in one place?

**Comment by behlendorf** [comment]
Yes, it looks like several new properties did end up in the wrong section.  All of the following should really be in the section for properties which can only be set at create time: casesensitivity, encryption, keyformat, normalization, pbkdf2iters, utf8only, volblocksize
